packages/edit-site/src/components/sync-state-with-url/use-init-edited-entity-from-url.js

```diff
@@ -111,15 +96,7 @@ function useResolveEditedEntityAndContext( { postId, postType } ) {
 					postTypeToResolve === 'page' &&
 					homepageId === postIdToResolve
 				) {
-					// We're still checking whether the front page template exists.
-					// Don't resolve the template yet.
-					if ( frontPageTemplateId === undefined ) {
-						return undefined;
-					}
-
-					if ( !! frontPageTemplateId ) {
-						return frontPageTemplateId;
-					}
+					return getDefaultTemplateId( { slug: 'front-page' } );
```

Comment:
Should we leave previous logic/step related comments? They should still apply, right?

Reply:
I don't think so? This returns the resolving selector right away so it will either return undefined or the ID. Worth noting it's already the same below for `postsPageId` which already returned a resolving selector before.